include/libdnf/comps/group/group.hpp

```diff
@@ -31,10 +32,11 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 namespace libdnf::comps {
 
+
 struct GroupId {
 public:
     GroupId() = default;
-    explicit GroupId(int id);
+    explicit GroupId(int id) : id(id){};
```

Comment:
Space before curled braces, not sure why clang format didn't catch this:
```c++
    explicit GroupId(int id) : id(id) {};
```

Reply:
clang-format doesn't think so :(